packages/svelte-query/src/context.ts

```diff
@@ -19,3 +21,17 @@ export const getQueryClientContext = (): QueryClient => {
 export const setQueryClientContext = (client: QueryClient): void => {
   setContext(_contextKey, client)
 }
+
+const _isRestoringContextKey = '$$_isRestoring'
+
+/** Retrieves a `isRestoring` from Svelte's context */
+export const getIsRestoringContext = (): Readable<boolean> => {
+  const isRestoring = getContext(_isRestoringContextKey)
+  if (!isRestoring) return readable(false)
+  return isRestoring as Readable<boolean>
+}
+
+/** Sets a `isRestoring` on Svelte's context */
+export const setIsRestoringContext = (isRestoring: Readable<boolean>): void => {
+  setContext(_isRestoringContextKey, isRestoring)
```

Comment:
Just want to check that this works correctly - is it okay to pass in a store directly into setContext like this?

Reply:
@aadito123 no that's awesome, I didn't realise it was a thing :)